### PR TITLE
added pbmm2 index and align commands

### PR DESCRIPTION
**Pull request by williamrowell**
pbmm2 is a wrapper for minimap2 supporting native PacBio formats for input and output.  I've added the pbmm2 index and align subcommands.

### CODE REVIEW

File: bio/pbmm2/align/test/Snakefile [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+        reference = "target/{reference}.fasta", # can be either genome index or genome fasta
+        query = "{query}.bam" # can be either unaligned bam, fastq, or fasta
+    output:
+        bam = "aligned/{query}.{reference}.bam",
+        index = "aligned/{query}.{reference}.bam.bai"
+    log:
+        "logs/pbmm2_align/{query}.{reference}.log"
+    params:
+        preset = "CCS",
+        sample = "",
```
Comment (johanneskoester) [inline bio/pbmm2/align/test/Snakefile:12]: Please explain each param with a comment to the right of it.

File: bio/pbmm2/align/test/Snakefile [hunk 2]
```diff
@@ -0,0 +1,16 @@
+rule pbmm2_align:
+    input:
+        reference = "target/{reference}.fasta", # can be either genome index or genome fasta
```
Comment (johanneskoester) [inline bio/pbmm2/align/test/Snakefile:3]: Please format all items as `key=...` instead of `key = ...`

Comment (johanneskoester) [reply bio/pbmm2/align/test/Snakefile:3]: same for the other wrapper below.

**Review by johanneskoester — CHANGES_REQUESTED** [review]
Thanks a lot! Just two minor things.

File: bio/pbmm2/align/test/Snakefile (hunk 1 above)
Comment (williamrowell) [reply bio/pbmm2/align/test/Snakefile]: Added comments for clarification.

File: bio/pbmm2/align/test/Snakefile (hunk 2 above)
Comment (williamrowell) [reply bio/pbmm2/align/test/Snakefile]: Reformatted both Snakefiles and pushed.  Thank you.

**Review by johanneskoester — APPROVED** [review]

**Pull request merged**